### ISSUE TITLE
deploy rasa-nlu-examples with docker 

### ISSUE THREAD

**Issue by yari0757** [RasaHQ/rasa-nlu-examples#47]
hi every one,
I trained a new just-nlu-model with my own w2v model. now, i want to put it in production/staging Env.
but official version of rasa docker not support this project. can i ask you show me how create a docker file from official rasa docker to use my own w2v model?

**Comment by sara-tagger** [comment]
Thanks for raising this issue, @dakshvar22 will get back to you about it soon✨

###### Please also check out the [docs](https://rasa.com/docs/) and the [forum](https://forum.rasa.com/) in case your issue was raised there too 🤗

**Comment by koaning** [comment]
I might look into that but odds are it will take a while. This repository currently only supports Rasa 1.10 , not Rasa 2.0.

One thing I am very curious about; what exactly did you pre-train :smile: ? Got something of a benchmark score that demonstrates your approach has merit to it?

**Comment by koaning** [comment]
Now that Rasa 2.0 support is done I can start thinking along here. 

What Dockerfile/setup do you currently have for your production environment? I might be able to write a guide but it helps to understand what exact flow you're using now.

**Comment by yari0757** [comment]
> I might look into that but odds are it will take a while. This repository currently only supports Rasa 1.10, not Rasa 2.0. We'd like to get Rasa 2.0 support in before we think about the docker containers.
> 
> One thing I am very curious about; what exactly did you pre-train ? Got something of a benchmark score that demonstrates your approach has merit to it?

I am using rasa nlu for NER and intent tagging but not for the dialog systems. I am using it for geocoding search. for example, user search in a pattern like  "city name+neighborhood+POI "or other formats, I am trying to tag entity in address and find user search intent(for example user search for a restaurant or city or street?).
my training data are generated by different patterns (not real data) and the pre-trained word embedding not working for my specific domain. I am trying to train my own w2v model. the accuracy of my models is improved but not too much! I wanna test my nlu model(which trained by w2v) by real user query to find out what happened on my metric.

for deploy rasa on production, i am using offical rasa docker image(full version) and pass model file by AWS s3 storage.
it should be great if you can write a docker file which support "nlu-example" project inside and we can pass a trained word embedding model by exactly same way AWS s3.

**Comment by koaning** [comment]
Are you running the service via `rasa`-cli or are you loading the rasa models via custom python code?

**Comment by yari0757** [comment]
> Are you running the service via `rasa`-cli or are you loading the rasa models via custom python code?
i am using rasa cli with officia docker image run on kubernetes by this command:
rasa run -m nlu-model.tar.gz --remote-storage aws --endpoints endpoints.yml --enable-api.

**Comment by koaning** [comment]
I've never run this repo in production before. As of yet I've only used it for benchmarking. 

That said, it should be as simple as adding a `pip install` to your `Dockerfile`. 

You might need to take heed with versions though. I'll only try to maintain the most recent version of Rasa here. That means that you might need to install a version that is tagged on an older Rasa release. Probably something like; 

```
pip install git+https://github.com/RasaHQ/rasa-nlu-examples/tree/0.1.3
```
